README.md

```diff
@@ -92,6 +92,14 @@ metadata:
   engine: true
 ```
 
+### RSpec Integration
+It's highly recommended that you add `--require stimpack/rspec` to your `.rspec` file to keep things simply.
+Or, if you'd like, pass it as an argument to `rspec`:
+
+```
+$ rspec --require stimpack/rspec ...
```

Comment:
Would we expect anyone to actually want to run rspec this way? I guess they could add that to a wrapper, just not sure why I'd want this.

Reply:
Not sure -- maybe CI?